esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -4472,6 +4475,48 @@ int CWsWorkunitsSoapBindingEx::onGetForm(IEspContext &context, CHttpRequest* req
     return onGetNotFound(context, request, response, service);
 }
 
+int CWsWorkunitsSoapBindingEx::onStartUpload(IEspContext &ctx, CHttpRequest* request, CHttpResponse* response, const char *serv, const char *method)
+{
+    StringArray fileNames, files;
+    StringBuffer source;
+    Owned<IMultiException> me = MakeMultiException(source.setf("WsWorkunits::%s()", method).str());
+    try
+    {
+        if (strieq(method, "ImportWUZAPFile"))
+        {
+            SecAccessFlags accessOwn, accessOthers;
+            getUserWuAccessFlags(ctx, accessOwn, accessOthers, false);
+            if ((accessOwn != SecAccess_Full) || (accessOthers != SecAccess_Full))
+                throw MakeStringException(-1, "Permission denied.");
+    
+            StringBuffer password, importQueryAssociatedFiles;
+            request->getParameter("importQueryAssociatedFiles", importQueryAssociatedFiles);
+            request->getParameter("Password", password);
```

Comment:
looks like this one was missed?

Reply:
@jakesmith the importQueryAssociatedFiles has been removed since it is never used. The 'Password' is capitalized because it is a part of the ESP request. It is read into the 'password' which is a c++ variable and is camelCase.

Reply:
@wangkx - ok understood.